### DISCUSSION TITLE
general question

### DISCUSSION THREAD

**Discussion by kimo095** [erodola/DLAI-s2-2022#13] [General]
Hello, good evening, my name is jebreel,  this is my first tutorial here, actually, I am not from a computer science background, my question is that it is enough for me the study the previous lectures in order to solve this tutorial.
thanks you

**Comment by erodola** [comment]
In general, you should be able to address all the notebooks and lectures if you first go through all the previous material. But I encourage you to do the notebooks in class as we go through the course, this way you can ask questions and compare with your colleagues.
